### ISSUE TITLE
Offline in local-directory

### ISSUE THREAD

**Issue by LaermTot** [klokantech/mapbox-gl-js-offline-example#3]
Hello

I tried using your demo locally by modifying the source a bit 

```
var idx = location.pathname.lastIndexOf("index.html");
var loc = location.origin+location.pathname;
if (idx > 0)
  loc = location.origin+location.pathname.substring(0,idx);

var style = {
  "version": 8,
  "sources": {
    "countries": {
      "type": "vector",
      "tiles": [loc+"/countries/{z}/{x}/{y}.pbf"],
      "maxzoom": 6
    }
  },
  "glyphs": loc+"font/{fontstack}/{range}.pbf",
  "layers": [{

```

the Problem was, that ocation.origin+location.pathname created a String like

file:///var/foo/bar/somewhere/**index.html ..** 

The Problem that I'm having tight now, is the fact that I simply get a Page with all the Navigation-Widgets but no map is being displayd .. 

Any hints what I'm doing wrong ?

.....

**Comment by bean-fidhleir** [comment]
Yes, you have to replace the location.* with the absolute path root.  I've no idea where the location.* vars are defined, but "index.html" is not the identifier of a subtree!  Which  means it can't be allowed in the middle of the path.

**Comment by hjrobinson** [comment]
I can't get the index.html to display my the vector tiles generated from mbutil.  The pbf files are in the folder called parcels_tiles. Is there a particular usage for a mbitles.pbf file (generated from mbutil) in this case? Here is a sample of my code:

var style = {
  "version": 8,
  "sources": {
    "tax_parcels": {
      "type": "vector",
      "tiles": ["file:///C:/parcels_tiles/{z}/{x}/{y}.pbf"],
    }
  },
  {
        "id": "tax_parcels",
        "type": "line",
        "source":"gig_harbor_parcels",
        "source-layer":"gig_harbor_parcels",
        "paint": {
          "line-color": "#bada55"
        }
      },
